server/ai_session.go

```diff
@@ -126,10 +124,17 @@ func (pool *AISessionPool) Remove(sess *BroadcastSession) {
 	delete(pool.sessMap, sess.Transcoder())
 	pool.inUseSess = removeSessionFromList(pool.inUseSess, sess)
 
-	// Magic number for now
-	penalty := 3
+	penalty := 0
 	// If this method is called assume that the orch should be suspended
-	// as well
+	// as well.  Since AISessionManager re-uses the pools the suspension
+	// penalty needs to consider the current suspender count to set the penalty
+	last_count, ok := pool.suspender.list[sess.Transcoder()]
+	if ok {
+		penalty = pool.suspender.count - last_count + pool.penalty
```

Comment:
I'm a little lost with this suspension logic. So, I see that:
- the `pool.suspender.count` is increased every time `signalRefresh()` is called
- `pool.penalty` is always set to `3`
- `last_count` is always set to `suspender.count + 3`

So, that logic would mean that we're not taking the suspended orchestrator until 3 times the `signalRefresh()` is called. Is this the idea of this suspension mechanism? That we don't allow the given O to get selected in the 3 refresh sessions?